src/main/java/Parser.java

```diff
@@ -0,0 +1,88 @@
+public class Parser {
+    public static Command processCommand(String command) {
+        boolean isList = command.equals("list");
+        if(isList) return Command.LIST;
+
+        boolean isBye = command.equals("bye");
+        if(isBye) return Command.BYE;
+
+        boolean isDone = command.equals("done");
+        if(isDone) return Command.DONE;
+
+        boolean isTodo = command.equals("todo");
+        if(isTodo) return Command.TODO;
+
+        boolean isDeadline = command.equals("deadline");
+        if(isDeadline) return Command.DEADLINE;
+
+        boolean isEvent = command.equals("event");
+        if(isEvent) return Command.EVENT;
+
```

Comment:
Is it better to use switch statement here?
```suggestion
        switch(command){
        case "list":
              return Command.LIST;
        case "bye":
             return Command.BYE;
             ....
        }

```